<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpWnd</class>
 <widget class="QWidget" name="SignUpWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1334</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="firstNameEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>230</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lastNameEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>270</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>310</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phoneNumberEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>350</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>430</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>550</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Don't have an account?</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>550</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 14px;</string>
   </property>
   <property name="text">
    <string>Log-in</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>510</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton{
	color: rgb(255, 255, 255);
	border-radius: 18px;
	background-color: rgb(57, 139, 247);
}
QPushButton::hover{
	background-color:#56bef5;
}	

</string>
   </property>
   <property name="text">
    <string>SIGNUP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="safeCodeEdit">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>470</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 12px;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Password reset code</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1334</width>
     <height>750</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/bg1.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>150</y>
     <width>331</width>
     <height>69</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 160);
font-size: 30px;</string>
   </property>
   <property name="text">
    <string>SIGN UP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="indent">
    <number>5</number>
   </property>
  </widget>
  <widget class="QPushButton" name="closedBtn">
   <property name="geometry">
    <rect>
     <x>1270</x>
     <y>10</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>firstNameEdit</zorder>
  <zorder>lastNameEdit</zorder>
  <zorder>emailEdit</zorder>
  <zorder>phoneNumberEdit</zorder>
  <zorder>userNameEdit</zorder>
  <zorder>passwordEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>commandLinkButton</zorder>
  <zorder>signupBtn</zorder>
  <zorder>safeCodeEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>closedBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
